src/frontend/src/flows/manage/deviceSettings.ts

```diff
@@ -329,3 +228,40 @@ const unprotectDevice = async (
   });
   back();
 };
+
+// Get a connection that's authenticated with the given device
+// NOTE: this expects a recovery phrase device
+const deviceConnection = async (
```

Comment:
this was moved down here; it was confusing to have as the first function in the module without context